### DISCUSSION TITLE
When building an absolutely clean project, it contains a css file with 500+ lines of code, where does it come from?!

### DISCUSSION THREAD

**Discussion by Semdevmaster** [nuxt/nuxt#17698] [Questions]
### Environment

------------------------------
- Operating System: `Windows_NT`
- Node Version:     `v16.14.2`
- Nuxt Version:     `3.0.0-rc.1`
- Package Manager:  `npm@8.3.0`
- Builder:          `vite`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`
------------------------------

### Reproduction

Only one file - app.vue, no css files or components
```
<template>
  <div>
    <h1>Hello world</h1>
  </div>
</template>
```

### Describe the bug

When you build the project, a css file appears in the .output > public > _nuxt folder,
containing more than 500 lines of code, as far as I understand, these are tailwindcss classes.
But there is not a single css file or component with styles in the project.

**Comment by danielroe** [comment]
It should be a single (long) line. It contains the CSS used for the default error/404 pages shipped by Nuxt. If you provide your own `~/error.vue` it should be empty.
